### ISSUE TITLE
Better error handling

### ISSUE THREAD

**Issue by sobolevn** [life4/flakehell#25]
Let's say we have `wemake-python-styleguide` installed (or just `flake8-builtins`). And we want to lint the following file:

```bash
» echo 'a, *(b, c) = [1, 2]' > ex.py
```

That would be a result: 

```
» flake8 ex.py                      
Traceback (most recent call last):
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/bin/flake8", line 10, in <module>
    sys.exit(main())
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/main/cli.py", line 18, in main
    app.run(argv)
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/main/application.py", line 393, in run
    self._run(argv)
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/main/application.py", line 381, in _run
    self.run_checks()
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/main/application.py", line 300, in run_checks
    self.file_checker_manager.run()
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/checker.py", line 331, in run
    self.run_serial()
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/checker.py", line 315, in run_serial
    checker.run_checks()
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/checker.py", line 598, in run_checks
    self.run_ast_checks()
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8/checker.py", line 502, in run_ast_checks
    for (line_number, offset, text, check) in runner:
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8_builtins.py", line 106, in run
    for line, offset, msg, rtype in value:
  File "/Users/sobolev/Documents/github/wemake-python-styleguide/.venv/lib/python3.7/site-packages/flake8_builtins.py", line 124, in check_assignment
    item.value.id in BUILTINS:
AttributeError: 'Tuple' object has no attribute 'id'
```

But, that's a very bad situation. I raised multiple issues in `flake8` itself, but it was not very successful. 

What to do instead? I guess we can:
1. Write a warning with the full information to the screen (or to the log file) and continue with other linters / files as usual
2. Raise a custom violation to indicate that there's a problem in this specific place

In other words: we should let user know that there's something wrong and tell exactly what's wrong. And then continue with the main task. 

I suppose, that this will be super useful right now, due to `python3.8` massive `ast` changes. There would be many new errors everywhere.

**Comment by orsinium** [comment]
What do you think about adding `--safe` flag? It will indicate to catch such exceptions and show a custom error (`F666` :upside_down_face: )

**Comment by sobolevn** [comment]
Nice! 👍
